app/src/ui/app.tsx

```diff
@@ -348,11 +348,64 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.openCurrentRepositoryInExternalEditor()
       case 'select-all':
         return this.selectAll()
+      case 'show-release-notes-popup':
+        return this.showFakeReleaseNotesPopup()
     }
 
     return assertNever(name, `Unknown menu event name: ${name}`)
   }
 
+  /**
+   * Show a release notes popup for a fake release, intended only to
+   * make it easier to verify changes to the popup. Has no meaning
+   * about a new release being available.
+   */
+  private showFakeReleaseNotesPopup() {
+    this.props.dispatcher.showPopup({
+      type: PopupType.ReleaseNotes,
+      newRelease: {
+        latestVersion: '42.7.99',
+        datePublished: 'Awesomeber 71, 2025',
+        pretext: 'kas',
```

Comment:
Not going to hold up this PR, but I'll cross-reference #5510 here which has the context about what this field represents.

Reply:
Thanks for the reference; I realize now what the `pretext` field represents.

But can you clarify what you meant by "not going to hold up this PR, but ..."? Was there something about #5510 that you'd ideally like to see addressed here? Even if I tackled that too, it seems to me like they are two distinct pieces of work that should have their own commits/PRs; this one being about showing the dialog, and that one being about the implementation of the dialog.

Reply:
No, I just wanted to add a link to this PR into #5510 so that someone can easily find this dev-mode option.